### PR TITLE
Update to Spanish

### PR DESCRIPTION
**Pull request by Gelraiser**
Update lang to Spanish

### CODE REVIEW

File: src/main/resources/assets/twilightforest/lang/es_ES.lang [hunk 1]
```diff
@@ -0,0 +1,608 @@
+#PARSE_ESCAPES
+
+itemGroup.twilightForest=Bosque Crepuscular
+
+advancement.twilightforest.desc = Entra en los misteriosos bosques mágicos: El Bosque Crepuscular
```
Comment (xbony2) [inline src/main/resources/assets/twilightforest/lang/es_ES.lang]: Uh, sup with these spaces before and after the =? You didn't use Yandex/Bing/Google Translate I don't think (I checked) but it's quite odd.

Comment (Gelraiser) [reply src/main/resources/assets/twilightforest/lang/es_ES.lang]: Use the translator that goes directly on the google page when you search for it and then try to adjust the translation to a more human Spanish. (like now).
Spaces are a problem?

Comment (Tamaized) [reply src/main/resources/assets/twilightforest/lang/es_ES.lang]: They are a problem yes, try testing this in game, you won't get the results you expect.

Comment (xbony2) [reply src/main/resources/assets/twilightforest/lang/es_ES.lang]: More just thought they looked ugly but that sounds like a bigger issue.

**Comment by Gelraiser** [comment]
The wrong spaces are eliminated.
Confirm that everything is correct and apologize for the inconvenience, I am new to this.

**Comment by xbony2** [comment]
Es bueno ^^

File: src/main/resources/assets/twilightforest/lang/es_ES.lang [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+twilightforest.config.loading_icon_wobble_bounce_frequency.tooltip=Frecuencia de oscilación y rebote.
+twilightforest.config.loading_icon_scale=Escala de iconos
+twilightforest.config.loading_icon_scale.tooltip=Escala del ícono completo de carga.
+twilightforest.config.loading_icon_bounciness=Icono Sin rebote
+twilightforest.config.loading_icon_bounciness.tooltip=Cuánto rebota el icono de carga.
+twilightforest.config.loading_icon_tilting=Icono Temblequeante
+twilightforest.config.loading_icon_tilting.tooltip=Cuánto se tambalea el icono de carga.
+twilightforest.config.loading_icon_tilt_pushback=Compensación de Inclinación de Íconos
+twilightforest.config.loading_icon_tilt_pushback.tooltip=Valor de retroceso para volver a centrar el tambaleo del icono de carga.
+twilightforest.book.lichtower.1=\ u00A78 [[El cuaderno de un explorador, mordido por monstruos]] \ u00A70 \ n \ nHe comenzado a examinar el extraño aura que rodea esta torre. Los ladrillos de la torre están protegidos por una maldición, más fuerte que cualquiera de los que he visto antes. La magia de la maldición
```
Comment (Tamaized) [inline src/main/resources/assets/twilightforest/lang/es_ES.lang:572]: There's a lot of weird spacing for the escapes here
`\ u00A78` as an example

**Comment by Tamaized** [comment]
Also note, the filename must be lowercase, as we use pack format 3
https://github.com/TeamTwilight/twilightforest/blob/1.12.x/src/main/resources/pack.mcmeta#L3

**Review by Tamaized — CHANGES_REQUESTED** [review]
Spacing issues need to be fixed still, filename needs to be lowercase, and these lang keys may be outdated now :stuck_out_tongue: could use another look over and update.

**Comment by Andromander** [comment]
As 1.12.x is no longer maintained, this PR will be closed.

If you wish to add you translation to the mod, you can edit the file so that it is up to date with 1.16.x and resubmit in another PR. Do note the outstanding change requests, such as correctly formatted file name, spacing issues, as well as the outdated lang keys.

**Pull request closed without merging**